### ISSUE TITLE
Malaceptor double bracket

### ISSUE THREAD

**Issue by The-Watcher-213** [BSData/wh40k-9e#11172]
This is the developer of Yellowscribe and I would just like to say the double Brackets when dealing with the Nid Malaceptor are causing Yellowscribe to not be able to parse that model and is causing a lot of problems. If you could fix this that would be great and if not I can figure out a work I am just requesting that could be brought into line with other units so that would it would not be as hard to parse if you do this it would be greatly appreciated. it is the double brackets in the wounds to my knowledge

**Comment by GenWilhelm** [comment]
When you say "double brackets," do you mean the naming scheme of `(1)`, `(2)`, `(3)`?

**Comment by curryjl** [comment]
I have been testing this out.  It appears that the formatting Yellowscribe is expecting is as follows:

`Maleceptor [2] (5-7 Wounds Remaining)`

After comparing with how all of the other races do it, this is consistent with those units as well.
